src/components/DashboardPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Card, Button, Container, Row, Col} from 'react-bootstrap';
+
+class Dashboard extends React.Component {
+  render() {
+    return (
+      <container>
+      <div>
+        <Container>
+          <Row>
+            <Col>
+      <Card className="card1">
+  <Card.Body>
+    <Button className="button" href="#" variant="primary">Add Recipe</Button>
```

Comment:
As we are using react router we should prevent reloading the app. Using href will reload the page. I suggest importing Link from "react-router-dom" and using this component within each button.